internal/controller/controllers/agentserviceconfig_controller.go

```diff
@@ -1177,6 +1178,7 @@ func newImageServiceStatefulSet(ctx context.Context, log logrus.FieldLogger, asc
 			{Name: "tls-certs", MountPath: "/etc/image-service/certs"},
 			{Name: "service-cabundle", MountPath: "/etc/image-service/ca-bundle"},
 			{Name: "image-service-data", MountPath: "/data"},
+			{Name: "additional-ca-bundle", MountPath: "/additional-ca-bundle"},
```

Comment:
Maybe put this in `/etc/image-service` like the others. How about `/etc/image-service/os-images-ca-bundle`?